WifiART/src/userinterface/ArtMvcController.java

```diff
@@ -27,20 +30,42 @@
      *
      * */
     public ArtMvcController(ArtMvcView fv, ArtMvcModel fm) {
-        this.fm = fm;
-        this.fv = fv;
+        this.model = fm;
+        this.view = fv;
 
         // Initialize GUI by copying state from model
-        this.fv.setViewState(fm.getState());
-        this.fm.setCallback(this);
+        this.view.setViewState(fm.getState());
+        this.model.setCallback(this);
 
         // TODO(dmevans) Add the listeners to the View here
+        view.setLciIncludedListener(actionEvent ->
+            model.getState().setLciIncluded(view.getLciIncluded()));
+        view.setZIncludedListener(actionEvent ->
+            model.getState().setZIncluded(view.getZIncluded()));
+        view.setUsageIncludedListener(actionEvent ->
+            model.getState().setUsageIncluded(view.getUsageIncluded()));
+        view.setBssidIncludedListener(actionEvent ->
+            model.getState().setBssidIncluded(view.getBssidIncluded()));
+        view.setLcrIncludedListener(actionEvent ->
+            model.getState().setLcrIncluded(view.getLcrIncluded()));
+        view.setBssidIncludedListener(actionEvent ->
+            model.getState().setLcrIncluded(view.getLcrIncluded()));
+
+
+
+        view.setInputFileNameListener(actionEvent ->
```

Comment:
Add a comment to explain why this items below the blank lines are grouped separately.